specs/README.md

```diff
@@ -1,84 +1,170 @@
+| Author                | Member of the Project Reunion community proposing a new API or change to an existing API                                                                 |
+| Design Representative | Member of the Windows API Design Representatives group responsible for the shape and direction of their component area within Windows or Project Reunion |
+| Reviewer              | Person providing commentary on an API under review                                                                                                       |
 
-* Create unsolicited PRs that add or modify  specs. To propose a new feature
-    please follow the Project Reunion contribution process [described here](../docs/contributor-guide.md)
-    and start by [opening a new issue](https://github.com/microsoft/ProjectReunion/issues/new/choose).
+For additional help, create a new `api-design` tagged issue and assign it to @jonwis for assistance.
 
-## Spec workflow
+### Authoring an API Spec
 
 For Project Reunion team members, the spec and API design workflow is:
 
 1. Ensure your proposal is approved and in plan.
 
-2. Create a new working branch ```user/<username>/<feature>``` or work in
-   your fork of the Project Reunion repo.
+2. Create a new working branch `user/<username>/<feature>` or work in your fork of the Project
+   Reunion repo.
+
+3. Create a new folder for your spec under the `specs` folder: `/specs/<feature>`
+
+4. Author your spec using the [spec template](spec_template.md). Please use relative links for
+   images or other assets in the folder. Be sure to read and follow the commented instructions in
+   the template.
+
+5. Work with your API design representative and iterate on the specification in your branch as much
+   as you see fit.
+
+When your spec is ready for review, your API representative will help guide you through the
+remainder.
+
+**DO** follow the authoring guidelines from [the contributor's guide](../docs/contributor-guide.md).
+
+**DO** follow the spec template guidance for authoring samples and IDL.
+
+**DO NOT** create a pull request from `user/<username>/<feature>` to `main` (see "Reviewing"
+section).
+
+### Reviewing APIs
+
+Your API representative will help you finish the design document and get it reviewed with members of
+the Windows API Design community. The process to finish the review is:
+
+1. The representative creates a new pull request from `user/<username>/<feature>` to the `main`
+   branch. Make sure the pull request mentions the original issue thread.
+
+2. The representative schedules time with the Windows API Design community to review the pull
+   request.
+
+    - As the design group is spread out we prefer giving no less than 48h notice when reviewing APIs
+    - Reviews are typically scheduled for Tuesdays 1-2pm PST/PDT or Thursdays 1-3pm PST/PDT
```

Comment:
Given the potential international audience, perhaps

```suggestion
    - Reviews are typically scheduled for Tuesdays 1-2pm or Thursdays 1-3pm (Seattle local time).
```

so people outside North America don't have to try to figure out what "PST" means. I mean, we're kind of lucky that Pitcairn Standard Time is also GMT-8.